README.md

```diff
@@ -309,6 +309,25 @@ If you want to disallow redirects, you can do it like this:
 page = MetaInspector.new('facebook.com', :allow_redirections => false)
 ```
 
+You can also customize how many redirects you wish to allow:
+
+```ruby
+page = MetaInspector.new('facebook.com', :faraday_options => { redirect: { limit: 5 } })
+```
+
+And even customize what to do in between each redirect:
+
+```ruby
+callback = proc do |previous_response, next_request|
+  ip_address = Resolv.getaddress(next_request.url.host)
+  raise 'Invalid address' if IPAddr.new(ip_address).private?
+end
+
+page = MetaInspector.new(url, faraday_options: { redirect: { callback: callback } })
+```
+
+You can make use of Faraday's other available options through the `faraday_options[:redirect]` Hash (see the available options [here](https://github.com/lostisland/faraday_middleware/blob/main/lib/faraday_middleware/response/follow_redirects.rb#L44)).
```

Comment:
Instead of this, I'd explain that the options passed in our `redirect` option, will be passed to the `FollowRedirects` middleware.

Reply:
Sure thing! 👍